src/docs/environment.mdx

```diff
@@ -164,3 +164,122 @@ If you would like to be able to run `devserver` outside of your root checkout, y
   <em>and</em> port are required in order for DNS and some pages urls to be
   displayed correctly.
 </Alert>
+
+
+## Troubleshooting
+
+**Problem:** something like `pkg_resources.DistributionNotFound: The 'some_dependency<0.6.0,>=0.5.5' distribution was not found and is required by sentry`
+
+**Solution:** Your virtualenv needs to be updated. In sentry, type `make install-sentry-dev`. In getsentry, type `make install-develop-requirements`.
+
+---
+
+**Problem:** You see `Error occured while trying to proxy to: dev.getsentry.net:8000/`
+
+**Solution:**  You likely need to upgrade your Python dependencies. Go to the git root directory and run `make install-sentry-dev`.
+
+---
+
+**Problem:**  "out of memory" issues after running the sentry or getsentry devserver, or during webpacking.
+
+**Solution:**
+
+```bash
+export NODE_OPTIONS="--max-old-space-size=4096"
```

Comment:
Additionally, we should plug direnv here. @dcramer wdyt?

Reply:
I think we should be leading to people to the paved roads. direnv seems to be working well for many people now. Using it lets us avoid the oom issue with node entirely and not have to worry about troubleshooting it ever again.

Reply:
Cool. I haven't seen anyone bitten by this for a while now, so you can just remove this then.